plugins/org.python.pydev/src/org/python/pydev/ui/dialogs/PyDialogHelpers.java

```diff
@@ -161,6 +170,42 @@ public static int openQuestionConfigureInterpreter(AbstractInterpreterManager m)
         return INTERPRETER_CANCEL_CONFIG;
     }
 
+    /**
+     * <p>Statically open a list dialog to choose an interpreter from Conda.</p>
+     * <p>It returns both the interpreter name and executable.</p>
+     * 
+     * @return NameAndExecutable
+     */
+    public static NameAndExecutable openCondaInterpreterSelection() {
+        List<String> envs = CondaPackageManager.listCondaEnvironments(PyDevCondaPreferences.getExecutable());
+        List<String> envNames = new ArrayList<String>();
+
+        for (String env : List.copyOf(envs)) {
+            envNames.add(env.substring(env.lastIndexOf('\\') + 1));
```

Comment:
In Linux this'd fail.

I think you should make `CondaPackageManager.listCondaEnvironments` return a `List<File>` and then use the file API to manipulate instead of doing string manipulations.